src/main.java

```diff
@@ -1,2 +1,7 @@
 public class main {
+    public static void main(String[] args) {
+        System.out.println("Hello World !");
```

Comment:
```suggestion
        System.out.println("Hello World!");
```
느낌표는 띄어쓰기하지 말고 붙여주세요